<ui version="4.0" >
 <class>FindWidget</class>
 <widget class="QWidget" name="FindWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QComboBox" name="comboFind" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2" >
    <widget class="QPushButton" name="findButton" >
     <property name="text" >
      <string>Find</string>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4" >
    <widget class="QRadioButton" name="radioForward" >
     <property name="text" >
      <string>Forward</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5" >
    <widget class="QRadioButton" name="radioBackward" >
     <property name="text" >
      <string>Backward</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="6" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>41</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QCheckBox" name="checkWords" >
     <property name="text" >
      <string>Whole words only</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QCheckBox" name="checkCase" >
     <property name="text" >
      <string>Case sensitive</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2" >
    <widget class="QCheckBox" name="checkBegin" >
     <property name="text" >
      <string>Start at Beginning</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5" >
    <widget class="QPushButton" name="closeButton" >
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
